tsconfig.json

```diff
@@ -4,6 +4,7 @@
         "target": "es6",
         "outDir": "out",
         "lib": [
+            "es2017",
```

Comment:
💡 This is added to avoid the `Object.values` - `compiler error`, more here : ) https://stackoverflow.com/questions/42966362/how-to-use-object-values-with-typescript